src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -388,6 +394,22 @@ public function newLine($count = 1)
     }
 
     /**
+     * Sets the input stream used by the SymfonyQuestionHelper.
+     *
+     * @param resource $stream
+     *
+     * @throws InvalidArgumentException If the given stream is not a resource
+     */
+    public function setInputStream($stream)
+    {
+        if (!is_resource($stream)) {
+            throw new InvalidArgumentException(sprintf('Input stream must be a valid resource, %s given', gettype($stream)));
```

Comment:
```diff
- throw new InvalidArgumentException(sprintf('Input stream must be a valid resource, %s given', gettype($stream)));
+ throw new InvalidArgumentException(sprintf('Input stream must be a valid resource, "%s" given', gettype($stream)));
```

(In Symfony, most exception messages use doubles quotes for values. At least it is used for FQCN, don't know about    value's type).